README.md

```diff
@@ -0,0 +1,27 @@
+# Sern CLI
+
+Our CLI is allowing you to create & streamline your Discord bot project without writing a single line of code!
+
+😁 **User Friendly Usage** <br>
+🔥 **At blazing Speeds** <br>
+🌱 **Efficiently** <br>
+
+## Installation
+
+```sh
+npm install @sern/cli@latest
+```
+
+```sh
+yarn add @sern/cli@latest
+```
+
+## Getting Started
+
+When you install the CLI, you can use our commands with **sern** prefix. Usage is simple;
+
+`sern <command> (opt)<flag>`
```

Comment:
add code block maybe?
```sh
sern <command> (opt)<flag>
```

Reply:
This is not an actual command, it's only a usage. Btw other things you requested will be here asap